recipes/globus-cli/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "globus-cli" %}
+{% set version = "1.9.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/globus/globus-cli/archive/{{ version }}.tar.gz
+  sha256: e577b1e53e5f2265fa1f08549e26c8bdf6dbc90c22722dd46277db08c13ac38e
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```

Comment:
[You need to specify `entry_points` here.](https://conda.io/projects/conda-build/en/latest/source/resources/define-metadata.html#python-entry-points)

Reply:
Okay, thanks.  I'll look into it.

Reply:
Thanks again for your help.  It should be taken care of now.